Gemfile

```diff
@@ -28,6 +28,10 @@ gem 'jbuilder', '~> 2.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
 
+# gem 'stripe', '>= 5.29.0'
```

Comment:
You can remove this line, we are already including this gem pointing to our GitHub repo.
Also, it may be good to have the lines toghether like:
```
gem 'solidus', git: 'https://github.com/emeritus-tech/solidus.git', ref: '14e4f8da1'
gem 'solidus_stripe', '~> 4.2'
gem 'solidus_auth_devise'
```